bluesky/protocols.py

```diff
@@ -151,26 +171,20 @@ def collect(self) -> Generator[Dict[str, Any], None, None]:
         """
         ...
 
-    def describe_collect(self) -> Dict[str, Configuration]:
+    # TODO: is this actually right? Do we need collect_asset_docs too?
+    def describe_collect(self) -> Dict[str, Dict[str, EventDescriptor]]:
         """This is like ``describe()`` on readable devices, but with an extra layer of nesting.
 
         Since a flyer can potentially return more than one event stream, this is a dict
         of stream names (strings) mapped to a ``describe()``-type output for each.
         """
         ...
 
-    # Remaining same as Readable
-    def read_configuration(self) -> Configuration:
-        """Same as for a Readable device."""
-        ...
-
-    def describe_configuration(self) -> Configuration:
-        """Same as for a Readable device."""
-        ...
-
 
 @runtime_checkable
 class Stageable(Protocol):
+    # TODO: we were going to extend these to be able to return plans, what
+    # signature should they have?
```

Comment:
@tacaswell we discussed returning plans but I'm not sure what the type signature should be

Reply:
defer this